duffel_api/models/offer.py

```diff
@@ -221,6 +221,32 @@ def from_json(cls, json: dict):
         )
 
 
+@dataclass
+class OfferSliceSegmentStop:
+    """Additional segment-specific information about the stops, if any, included in the segment"""
+
+    id: str
+    duration: str
```

Comment:
**test:** Similar request here to add some stops to the test fixture [1].

[1] https://github.com/duffelhq/duffel-api-python/blob/3f01a35833a40a87b242d5328890491531ffcc04/tests/fixtures/get-offers.json#L135-L208